connectors/embeddings-vector-database/element-templates/hybrid/embeddings-vector-database-outbound-connector-hybrid.json

```diff
@@ -1239,7 +1431,7 @@
     },
     "type" : "String"
   }, {
-    "id" : "vectorStore.azureCosmosDbNoSql.consistencyLevel",
+    "id" : "vectorStore.consistencyLevel",
```

Comment:
🔧 Is this what we expect it to be? 🤔 As this property is cosmos DB specific, I would assume the property to contain cosmos DB in the ID.

Regarding the discussion in https://camunda.slack.com/archives/C02JLRNQQ05/p1754407737998059: assuming there would be another vector store configuring a consistency level, would that still work?